src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs

```diff
@@ -530,5 +530,57 @@ public static string Page(this IUrlHelper urlHelper, string pageName, object val
                 host: host,
                 fragment: fragment);
         }
+
+        /// <summary>
+        /// Generates an absolute URL for an action method, which contains the specified
+        /// <paramref name="action"/> name, <paramref name="controller"/> name, route <paramref name="values"/>,
+        /// <paramref name="protocol"/> to use, <paramref name="host"/> name, and <paramref name="fragment"/>.
+        /// Generates an absolute URL if the <paramref name="protocol"/> and <paramref name="host"/> are
+        /// non-<c>null</c>. See the remarks section for important security information.
+        /// </summary>
+        /// <param name="helper">The <see cref="IUrlHelper"/>.</param>
+        /// <param name="action">The name of the action method. When not <see langword="null" />, defaults to the current executing action.</param>
+        /// <param name="controller">The name of the controller. When not <see langword="null" />, defaults to the current executing controller.</param>
```

Comment:
should be *“when `null`, defaults to …”*

Reply:
Yeah, just edited my comment 😦